quixstreams/platforms/quix/api.py

```diff
@@ -63,24 +61,47 @@ def request(self, method, url, **kwargs):
     @property
     def default_workspace_id(self) -> str:
         if not self._default_workspace_id:
-            raise self.UndefinedQuixWorkspaceId("You must provide a Quix Workspace ID")
+            raise UndefinedQuixWorkspaceId("You must provide a Quix Workspace ID")
```

Comment:
Let's specify how exactly it should be provided (e.g. via env variable), because we don't accept workspace IDs in `Application`.